internal/metrics/tx_mapper_db.go

```diff
@@ -412,6 +412,12 @@ func (tm *TxMapperDB) processTransactionExecution(
 			}
 			continue
 		}
+
+		log.Info().Uint64("gas", decryptedTx.Gas()).
```

Comment:
can you please check that this is legacy and dynamic tx type compatble?

Reply:
yes both types follow the interface
https://github.com/ethereum/go-ethereum/blob/16bf471151a2e3c65499e0e9ae913e31d634826e/core/types/tx_legacy.go#L99
https://github.com/ethereum/go-ethereum/blob/16bf471151a2e3c65499e0e9ae913e31d634826e/core/types/tx_dynamic_fee.go#L92